server/blob/blob-union/src/test/java/org/apache/james/blob/union/UnionBlobStoreTest.java

```diff
@@ -50,23 +49,25 @@
 import org.testcontainers.shaded.com.google.common.base.MoreObjects;
 import org.testcontainers.shaded.com.google.common.collect.ImmutableList;
 
+import reactor.core.publisher.Mono;
+
 class UnionBlobStoreTest implements BlobStoreContract {
 
     private static class FutureThrowingBlobStore implements BlobStore {
```

Comment:
This class name may does not make sense now. Cause we use `Mono` to replace `CompletableFuture`

Reply:
I changed to FailingBlobStore